app/assets/javascripts/template_invocation.js

```diff
@@ -0,0 +1,32 @@
+$(function() { job_invocation_form_binds() });
+
+function refresh_execution_form() {
```

Comment:
luckily there's no such convention in Foreman - https://github.com/theforeman/foreman/blob/develop/app/assets/javascripts/application.js

could we keep it this way?